ksp/src/main/kotlin/net/skyscanner/backpack/ksp/visitor/ViewStoriesVisitor.kt

```diff
@@ -0,0 +1,42 @@
+package net.skyscanner.backpack.ksp.visitor
+
+import com.google.devtools.ksp.symbol.FileLocation
+import com.google.devtools.ksp.symbol.KSFunctionDeclaration
+import com.google.devtools.ksp.symbol.KSName
+import com.google.devtools.ksp.symbol.KSNode
+import com.google.devtools.ksp.visitor.KSDefaultVisitor
+import net.skyscanner.backpack.ksp.ComponentDefinition
+import net.skyscanner.backpack.ksp.StoryDefinition
+import net.skyscanner.backpack.ksp.ViewStoryAnnotation
+import net.skyscanner.backpack.ksp.find
+import net.skyscanner.backpack.ksp.get
+
+object ViewStoriesVisitor : KSDefaultVisitor<Map<KSName, ComponentDefinition>, StoryDefinition?>() {
```

Comment:
looks like the view/compose classes are almost the same - can some bits be extracted?

Reply:
I was thinking about that, but it'll make the whole stuff more complicated to read and maintain. And it's just 2 classes

Reply:
the two need to be in sync, though, so this feels harder to maintain?